modules/apps/message-boards/message-boards-api/src/main/java/com/liferay/message/boards/exception/BoardAttachmentException.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.message.boards.exception;
+
+import com.liferay.portal.kernel.exception.NoSuchModelException;
+
+/**
+ * @author Brian Wing Shun Chan
+ */
+public class BoardAttachmentException extends NoSuchModelException {
```

Comment:
Why is this called `BoardAttachmentException`? Shouldn't it be called `MessageBoardsAttachmentException`?

Reply:
As I told to Carlos, I just called `BoardAttachmentException` because in the last issue (WikiPageAttachment) I've created a new one called `WikiPageAttachmentException` , but Brian told me that it should be named as `PageAttachmentException`. That's why I called it in that way

Reply:
Sounds weird, can you double check with the Lima team?

Reply:
See here @4lejandrito: https://github.com/liferay/liferay-portal/blob/master/modules/apps/wiki/wiki-service/service.xml#:~:text=%3Cexception%3E-,PageAttachment,-%3C/exception%3E

Reply:
However, I'm going to send this PR to lima team and check what they think :+1: